db/migrate/20190604172348_add_columns_to_users.rb

```diff
@@ -1,6 +1,6 @@
 class AddColumnsToUsers < ActiveRecord::Migration[5.2]
   def change
     add_column :users, :remember_created_at, :timestamp
-    add_column :users, :avatar_url, :string
+    rename_column :users, :pic_url, :avatar_url
```

Comment:
Can this change be a new migration that removes the pic_url column? Usually not a good idea to edit existing migrations.

Reply:
I was thinking that since we didnt have real data yet, it wouldnt matter.  @markdamobi  Did you mean just create a new migration that removes the pic_url first then add avatar_url or just  new migration that renames it?

Reply:
@gcwokeup right now on the master branch, there's both pic_url and avatar_url. Looks like avatar_url is what's being populated in the seed file. So I think you just need a migration to delete the pic_url. Unless I'm misunderstanding what you're trying to do.

Reply:
> I was thinking that since we didnt have real data yet, it wouldn't matter.
You're right, it shouldn't matter much if it's not real data right now. But then whoever pulls in the changes would have to re-create their database before getting those changes. Which isn't really a big deal right now, but just something to keep in mind.

Reply:
@markdamobi cool i get you. ill do a remove on pic_url. thanks

Reply:
k. i moved it to a separate migration